cmd/minishift/cmd/start.go

```diff
@@ -274,11 +291,25 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		ShellProxyEnv:    shellProxyEnv,
 	}
 
-	fmt.Printf("Starting local OpenShift cluster using '%s' hypervisor...\n", machineConfig.VMDriver)
+	fmt.Printf(" using '%s' hypervisor ...\n", machineConfig.VMDriver)
 	var hostVm *host.Host
+
+	// configuration with these settings only happen on create
+	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
+	if !isRestart {
+		fmt.Println("-- Minishift VM will be configured with ...")
+		fmt.Println("   Memory:    ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   vCPUs :    ", machineConfig.CPUs)
+		fmt.Println("   Disk size: ", units.HumanSize(float64(machineConfig.DiskSize*units.MB)))
```

Comment:
Possible, but... and this is my personal opinion: Lingering in a quest for perfection is not ideal. This PR has been open for already 12 days now... although mostly waiting for comments. But now, most of the current suggestions are nitpicking about function names, which have not much to do with the actual functionality and adding new requests (although small). This is mostly slowing integration down and allows less time to expose it to general testing/usage.

I believe these can be handled as part of https://en.wikipedia.org/wiki/Code_refactoring. Note: Note: we unnecessarily made refactoring harder as we want each commit referring to an issue number?

My 2 BCC cents